### PR TITLE
Checkout contribution: changes to memory.md

### PR DESCRIPTION
**Pull request by karenword**
I'm a member of The Carpentries Core Team and I'm submitting this issue on behalf of another member of the community. In most cases, I won't be able to follow up or provide more details other than what I'm providing below.

---------------------------------------------------

Hi there, 
I have updated the flow of the instruction as below as in my view the structure of the course lacks coherence at some stages.

There are 2 cognitive resources, memory and attention.
- Memory - details and how to manage memory
- Attention- details and how to manage attention

Overloading either of the creates Cognitive overload.
- Cognitive load and discuss strategies to manage cognitive overload.

Happy to discuss and explain further.

Regards,
Doschmund Kwiatkowski

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

---

### CODE REVIEW

File: episodes/05-memory.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ In our final topic in how people learn (and therefore, how we can be more
 effective instructors), we will be learning more about human memory:
 specifically, how to remove unnecessary "load" in order to facilitate learning.
 
-::::::::::::::::::::::::::::::::::::::: objectives
+::::::::::::::::::::::::::::::::::::::: 
+objectives
```
Comment (zkamvar) [inline episodes/05-memory.md:13]: Somehow the fenced div got severed here. This should fix it. 

```suggestion
::::::::::::::::::::::::::::::::::::::: objectives
```

Comment (zkamvar) [reply episodes/05-memory.md:13]: @KwiatkowskiD here is a guide on how you can update your PR with this suggestion: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by karenword** [comment]
This cannot be merged as-is or maintainer-edited, but I think it's a useful idea. @KwiatkowskiD let us know if you'd like to work with us to address feedback; otherwise we can close and create an issue or alternate PR.

**Comment by KwiatkowskiD** [comment]
Hi Karen,

I do not use GitHub often.
I am unsure what is expected from me on this.
Let me know how I can help.

Regards,
Doschmund

**Comment by karenword** [comment]
Hi Doschmund,

Thanks for your reply! Nothing is expected! However, if you would like to take this opportunity to try some practice with GitHub, we would welcome your contribution. If it can be fixed, you will be recorded as a co-author on the next curriculum release. If not, I will submit a new PR that acknowledges you, but our automated systems will not note your contribution for authorship.

If you would like to try to fix it, you can view comments in the "files changed" tab when viewing this PR in your browser. You can then edit the file (look for the three dots at the top right of the file window) to make the suggested changes.

If you would rather we create a new PR with your idea, I'd be happy to do that. Just let us know!

Best,

Karen

**Pull request closed without merging**